src/components/BadgeSelect.tsx

```diff
@@ -79,16 +80,38 @@ class BadgeSelect extends React.Component<BadgeSelectProps, BadgeSelectState> {
 					</Text>
 				}
 			>
+				{!!this.props.extraPrefix &&
+					this.props.extraPrefix.map(item => (
+						<ButtonWrapper key={item} onClick={() => this.setSelected(item)}>
+							<Text fontSize={2}>{item}</Text>
+						</ButtonWrapper>
+					))}
+
+				{!!this.props.extraPrefix && (
```

Comment:
There's a subtle tiny sort-of bug here (and below): if I set `extraPrefix: []` then I'll get a divider, because `!![] === true`

Reply:
I'll create an issue and resolve it at a later date: https://github.com/resin-io-modules/rendition/issues/356